docs/user/dql/functions.rst

```diff
@@ -2037,6 +2038,60 @@ Example::
     | 13:49:00           | 13:49:00        | 13:49:00                                 | 13:49:00                      |
     +--------------------+-----------------+------------------------------------------+-------------------------------+
 
+TIME_FORMAT
+-----------
+
+Description
+>>>>>>>>>>>
+
+Usage: time_format(time, format) formats the time argument using the specifiers in the format argument.
+This supports a subset of the time format specifiers available for the `date_format`_ function.
+Using date format specifiers supported by `date_format`_ will return 0 or null.
+Acceptable format specifiers are listed in the table below.
+If an argument of type DATE is passed in, it is treated as a DATETIME at midnight (i.e., 00:00:00).
+
+.. list-table:: The following table describes the available specifier arguments.
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Specifier
+     - Description
+   * - %f
+     - Microseconds (000000..999999)
+   * - %H
+     - Hour (00..23)
+   * - %h
+     - Hour (01..12)
+   * - %I
+     - Hour (01..12)
+   * - %i
+     - Minutes, numeric (00..59)
+   * - %p
+     - AM or PM
+   * - %r
+     - Time, 12-hour (hh:mm:ss followed by AM or PM)
+   * - %S
+     - Seconds (00..59)
+   * - %s
+     - Seconds (00..59)
+   * - %T
+     - Time, 24-hour (hh:mm:ss)
+
+
+Argument type: STRING/DATE/DATETIME/TIME/TIMESTAMP
```

Comment:
Second arg is string

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/204/commits/c44602293107b2037d35f9c09138451a73c3bff1